pkg/osbuild/skopeo_source.go

```diff
@@ -47,6 +54,12 @@ func (item SkopeoSourceItem) validate() error {
 		return fmt.Errorf("source item %#v has invalid digest", item)
 	}
 
+	if transport := item.Image.ContainersTransport; transport != nil {
+		if *transport != DockerTransport && *transport != ContainersStorageTransport {
```

Comment:
I wonder if a common helper like `validateTransport` could be extracted here? I think the same code is used `pkg/osbuild/skopeo_index_source.go` ?